setup.py

```diff
@@ -10,6 +11,10 @@
   'dataframe': ['numpy', 'pandas >= 0.16.0', 'toolz >= 0.7.2', 'partd >= 0.3.2'],
   'imperative': ['toolz >= 0.7.2'],
 }
+if (sys.version_info[0] < 3
+        or sys.version_info[0] == 3 and sys.version_info[0] < 3):
+    extras_require['lzma'] = ['backports.lzma >= 0.0.3']
```

Comment:
I'm not sure if this is something we'd want to do here or not. Currently our only extras are subpackages (`dask.dataframe`, etc...). There are many features in dask that use optional dependencies (for example, `graphviz` is required for displaying dask graphs), but we don't include them here in `extras_require`.